concent-builder/role-tiller.yml

```diff
@@ -0,0 +1,49 @@
+# Tiller permission to all resources in default namespace
+kind: Role
+apiVersion: rbac.authorization.k8s.io/v1beta1
+metadata:
+  name: tiller-manager
+  namespace: default
+rules:
+- apiGroups: ["", "extensions", "apps"]
+  resources: ["*"]
+  verbs: ["*"]
+---
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1beta1
+metadata:
+  name: tiller-binding
```

Comment:
The name should clearly say what is being bound to what.

Maybe `default-namespace-manager-to-tiller`?

Reply:
Done.